internal/pipeline/executor.go

```diff
@@ -138,9 +140,19 @@ func (p *pkg) resolveSources() []string {
 			// include only this pkg sources
 			pkgPaths = append(pkgPaths, p.Path())
 		case sourceAllSubPkgs:
-			// TODO(droot): figure out how to enumerate sub packages for current pkgPath.
-			// For now including current pkg
+			// For now including current pkg and subpkgs in current directory
 			pkgPaths = append(pkgPaths, p.Path())
+			files, _ := ioutil.ReadDir(p.Path())
```

Comment:
Change the signature of the `resolveResources` to return this error. Don't ignore this error :)

Reply:
done